mongo/options/clientoptions.go

```diff
@@ -111,12 +128,17 @@ func Client() *ClientOptions {
 // Validate validates the client options. This method will return the first error found.
 func (c *ClientOptions) Validate() error { return c.err }
 
-// ApplyURI parses the provided connection string and sets the values and options accordingly.
+// ApplyURI parses the given URI and sets options accordingly. The URI can contain host names, IPv4/IPv6 literals, or
+// an SRV record that will be resolved when the Client is created. When using an SRV record, TLS support is
+// implictly enabled. Specify the "tls=false" URI option to override this. If the connection string contains any options
```

Comment:
Let's also specify the inverse: 

    If an option is set on ClientOptions after this method is called, that option will override any option applied via the connection string